src/TestUtils/src/DeviceTests/AssertionExtensions.Windows.cs

```diff
@@ -231,6 +231,7 @@ public static async Task<T> AttachAndRun<T>(this FrameworkElement view, Func<Win
 
 				try
 				{
+					await Task.Delay(50);
```

Comment:
Thi was the real fix since the switch animates to on. This may also fix a few other flakey tests that needed a few extra ms to get settled.

Reply:
Is 50 enough to wait for an animation here? is that going to be flaky still a bit?  Would bumping this to 100-250ms be a huge impact on the total test time?

Reply:
Made it 100 for now.